graylog2-server/src/main/java/org/graylog2/rest/resources/system/logs/LoggersResource.java

```diff
@@ -151,6 +159,48 @@ public void setSingleLoggerLevel(
         logger.setLevel(newLevel);
     }
 
+    @GET
+    @Timed
+    @ApiOperation(value = "Get recent log messages")
+    @ApiResponses(value = {
+            @ApiResponse(code = 404, message = "Memory appender is disabled or of wrong instance.")
+    })
+    @Path("/messages/recent")
+    @Produces(MediaType.APPLICATION_JSON)
+    public LogMessagesSummary messages(@ApiParam(name = "limit", required = true, defaultValue = "500") @QueryParam("limit") int limit) {
```

Comment:
Unfortunately that's not enough, as `@ApiParam` is just an annotation used by Swagger to generate the documentation/Swagger specification. `@DefaultParam` is the correct annotation. :trollface: